web-playground/jest.config.js

```diff
@@ -1,6 +1,6 @@
-module.exports = {
+export default {
     preset: "jest-puppeteer",
     transform: {
         "^.+\\.ts?$": "ts-jest"
     },
-};
+  }
```

Comment:
Needs a newline

Reply:
Fixed.

Reply:
It wasn't quite, so I force pushed a fix.

Reply:
Oh, I checked in indenting on the last line 🤦‍♂️

Maybe I should start using pre-commit hooks.